nservicebus/gateway/scale-out.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gateway And Scale-Out
+summary: Explanation of how the Gateway handles Scale-Out
+tags: []
+related:
+ - samples/gateway
+---
+
+The gateway is designed to run on a single instance for each logical endpoint. A scaled out endpoint needs one instance to receive all the incoming HTTP gateway traffic for that endpoint. There are different strategies to achieve this based on the transport and the scale-out mechanism utilized:
+
+### MSMQ with the distributor
+
+With MSMQ scale out using the [distributor](/nservicebus/scalability-and-ha/distributor), the gateway runs on the master node/distributor, making it the natural single receiving gateway instance for the endpoint. Running the master node/distributor as [a clustered service on a Windows Failover Cluster](/nservicebus/scalability-and-ha/deploying-to-a-windows-failover-cluster.md) provides High Availability(HA). No special action must be taken other than configuring senders to send gateway HTTP traffic to the master node instead of directly to the workers.
```

Comment:
`When scaling out MSMQ through the distributor, the node running the distribytor is a natural candidate to also run the gateway` ?